package.json

```diff
@@ -61,7 +61,7 @@
     "body-parser": "1.18.3",
     "dateformat": "3.0.3",
     "express": "~4.16.4",
-    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "iotagent-node-lib": "git://github.com/Madhu-NEC/iotagent-node-lib.git#selectively_ignore_some_measure",
```

Comment:
To be revered once PR https://github.com/telefonicaid/iotagent-node-lib/pull/835 gets merged (just a remark note).